src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -169,22 +166,9 @@ public SearchResultView searchNew(@PathVariable("jid") final String jurisdiction
 
         Map<String, String> sanitized = fieldMapSanitizeOperation.execute(params);
 
-        addSortField(metadata, sanitized);
-
         return searchQueryOperation.execute(view, metadata, sanitized);
     }
 
-    private void addSortField(MetaData metadata, Map<String, String> queryParameters) {
-        //Some (ugly) hardcoding (RDM-4636), until we provide feature for default sorting of search results via definition
-        if (CASE_TYPE_DIVORCE.equalsIgnoreCase(metadata.getCaseTypeId())) {
-            if (queryParameters.isEmpty()) {
-                metadata.setSortField(CASE_DATA_ENTITY_FIELD_LAST_MODIFIED);
```

Comment:
is `sortField` in Metadata and it's accessory methods still needed or we can remove? I can't see it used any longer. More in general, can you look at RDM-4636 changes (including tests) and remove everything that is no longer needed please?

Reply:
It's used in getQueryByMetaData(MetaData metadata).. so will go away once that's removed.

Reply:
actually removed `sortField ' now it self 😃